mmv1/products/identityplatform/Config.yaml

```diff
@@ -72,6 +102,36 @@ properties:
     name: 'autodeleteAnonymousUsers'
     description: |
       Whether anonymous users will be auto-deleted after a period of 30 days
+  - !ruby/object:Api::Type::NestedObject
+    name: 'multiTenant'
+    description: |
+      Configuration related to multi-tenant functionality.
+    properties:
+      - !ruby/object:Api::Type::Boolean
```

Comment:
Can we have some test coverage on negative value for boolean fields. Just want to make sure the API returns the value back if we explicitly set the `false` value, otherwise, it will cause a permadiff and we may need to add a custom flattener to handle it